inc/actions.php

```diff
@@ -224,7 +224,8 @@ function dsi_eventi_filters( $query ) {
  */
 function dsi_schede_progetti_filters( WP_Query $query ) {
 
-    if ( ! is_admin() && $query->is_main_query() && (is_post_type_archive("scheda_progetto") || (get_queried_object()?->taxonomy ?? null) == "tipologia-progetto") ) {
+    if ( ! is_admin() && $query->is_main_query() && (is_post_type_archive("scheda_progetto") || (get_queried_object() && isset(get_queried_object()->taxonomy) && get_queried_object()->taxonomy == "tipologia-progetto")) ) {
```

Comment:
Segnalo che WP ha anche una funzione _[is_tax()](https://developer.wordpress.org/reference/functions/is_tax/)_, che è più compatta e "idiomatica" e svolge già tutti i controlli su _get_queried_object()_.
```suggestion
    if ( ! is_admin() && $query->is_main_query() && (is_post_type_archive("scheda_progetto") || is_tax("tipologia-progetto")) ) {
``` 
[v. #691]

Reply:
qui si è modificata la linea precedente, in quanto la versione presente andava in conflitto con versioni di PHP precedenti all'8.0

Reply:
Sì, infatti: è l'operatore _nullsafe_ `?->`, che non è supportato.
Ma di tutta la sequenza delle verifiche su `get_queried_object->taxonomy` credo che se ne potrebbe fare a meno, perché quello è proprio il mestiere di `is_tax()`.